File: 1 - Sequenciais/sequenciais6.java
//Faça um programa que peça o raio de um círculo, calcule e mostre sua área.

import java.util.Scanner;

public class sequenciais6 {

  public static void main(String[] args) {
    System.out.println("Digite o raio da circunferencia: ");
    float raio = new Scanner(System.in).nextFloat();
    double area = Math.PI * Math.pow(raio, 2);
    System.out.printf("A area da circunferencia eh de: %.2f", area);
  }

}

File: 1 - Sequenciais/Ruby/sequenciais6.rb
#Faça um programa que peça o raio de um círculo, calcule e mostre sua área.

puts "Digite o raio da circunferência: "
raio = gets.to_f
area = (Math::PI * (raio ** 2)).round(2)
puts "A área da circunferencia é de: #{area}"
